Makefile

```diff
@@ -55,11 +45,12 @@ build: binary-build
 	@$(call check-static-binary,deis)
 
 build-all:
-	${DEV_ENV_CMD} gox -verbose \
-	${GO_LDFLAGS} \
-	-os="linux darwin " \
-	-arch="amd64 386" \
-	-output="$(DIST_DIR)/deis-${VERSION}-{{.OS}}-{{.Arch}}" .
+	${DEV_ENV_CMD} gox -verbose ${GO_LDFLAGS} -os="linux darwin " -arch="amd64 386" -output="$(DIST_DIR)/deis-latest-{{.OS}}-{{.Arch}}" .
```

Comment:
To clarify my understanding, this and the two commands below will:

1. update/override the existing `deis-latest-{{.OS}}-{{.Arch}}` binary with this latest version 
2. throw latest version into the `${TRAVIS_TAG}` subdir <--- What if this env var is not set/does not exist?  (It will only exist if the travis build is kicked off from pushing a git tag to the repo, right?)
3. throw latest version into the `${VERSION}` subdir

As long as I'm understanding correctly, my only real question is in 2. above...